app/src/main/res/layout/fragment_alert_allergens.xml

```diff
@@ -6,7 +6,8 @@
     <android.support.v7.widget.RecyclerView
         android:id="@+id/allergens_recycle"
         android:layout_width="match_parent"
-        android:layout_height="match_parent" />
+        android:layout_height="match_parent"
+        android:paddingBottom="100dp"/>
```

Comment:
As far as I am aware, you would be better using `marginBottom` here as it adds space after the bottom item and lets the view reach down to the bottom rather than having it "sit" on top of the floating action button